network/servicenetwork/suite_test.go

```diff
@@ -234,6 +251,27 @@ func (s *testSuite) TearDownTest() {
 	s.fixture().pulsar.Stop(s.fixture().ctx)
 }
 
+func waitFileToBeRemoved() {
+	var fileMarker = "DELETE_THIS_FILE_FOR_CONTINUE"
+	file, err := os.OpenFile(fileMarker, os.O_RDONLY|os.O_CREATE, 0600)
+	if err != nil {
+		log.Warn("Cannot create file " + fileMarker)
+		return
+	}
+	for fileExists(file.Name()) {
+		time.Sleep(time.Second * 5)
+		log.Info("Tests are suspended. Delete file to continue. " + fileMarker)
```

Comment:
This will produce thousands of unneeded log lines

Reply:
increased to 60 sec